### DISCUSSION TITLE
Why the prevalence of alt keybindings

### DISCUSSION THREAD

**Discussion by rcorre** [helix-editor/helix#5824] [Q&A]
I've noticed helix chooses to use a lot of "Alt-<key>" bindings, even in cases where "Ctrl-<key>" is unbound.
All the applications I use daily primarily use "Ctrl" as a modifier. I hardly need "Alt" for anything outside helix.
That makes this seem like an unusual design decisions, but I checked the FAQ, docs, and discussions, and didn't find an explanation.

I'm curious for a few reasons:

- Since I always need `ctrl` and almost never use `alt`, I use a keyboard layout where `ctrl` is very easy to reach and `alt` is hard to reach, which works well for most applications
- My terminal (`st`) does not appear to send `alt` to applications. Maybe there's a way to fix this, but a cursory internet search is failing me, and it's not something I've ever needed before.
- New features seem to favor adding more `alt` keybinds, e.g. https://github.com/helix-editor/helix/pull/4189 (which matters because picker keybinds currently are not mappable).

**Comment by ni-be** [comment]
Sure there are some bindings use ALT but most that do also have alternatives. But I share your sentiment I had to remove / change some of them because I use Zellij / Multiplexer the same way I use a Tiling window WM but instead of Super as mod I use Alt for Zellij so there were 6 conflicts.
